README.md

```diff
@@ -357,7 +357,7 @@ Since Panther implements the API of popular libraries, it already has an extensi
 
 The following environment variables can be set to change some Panther's behaviour:
 
-* `PANTHER_NO_HEADLESS`: to disable the browser's headless mode (will display the testing window, useful to debug)
+* `PANTHER_NO_HEADLESS`: if defined and true, then it disables Panther's default use of headless mode with a 1200x1100 window (and disable GPU for chrome) 
```

Comment:
this is documenting internal details that might change (see eg #640) so I'm not sure about the benefit of stating this :thinking: 

```suggestion
* `PANTHER_NO_HEADLESS`: set to `1` to disable Panther's use of headless mode and thus see what happens in a browser window
```